jenkins/ci.opensuse.org/openstack-prepare-staging.yaml

```diff
@@ -0,0 +1,148 @@
+- job:
+    name: 'openstack-prepare-staging'
+    node: cloud-trackupstream
+
+    parameters:
+      - string:
+          name: openstack_project
+          default: None
+          description: name of the main project that should be updated with packages from its staging project
+
+    builders:
+      - shell: |
+          #!/usr/bin/python2
```

Comment:
this might be starting to nitpick, but I would prefer that as a script to be called and not inline the jenkins job. this allows also self-testing and manual debugging (or running tests against the code with a separate testcase collection).

Reply:
Done.